src/components/RadioButton/__stories__/RadioButton.stories.tsx

```diff
@@ -14,10 +14,12 @@ const options: RadioButtonOption[] = [
     {value: 'Value 3', content: 'Value 3'},
 ];
 
-const DefaultTemplate: Story<RadioButtonProps> = (args) => <RadioButton {...args} />;
+const DefaultTemplate: Story<RadioButtonProps> = (args) => {
+    const [value, setValue] = React.useState<string>(options[0].value);
```

Comment:
Why not use storybook controls?

Reply:
Nope, storybook is not intended to be interactive, it is more like UI snapshots than demos.

Reply:
It's kinda weird when you open a demo and it's not working properly. And the only way to change the value is to do it by hand.